packages/knip/test/plugins/changelogen.test.ts

```diff
@@ -0,0 +1,22 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/changelogen');
+
+test('Find dependencies with the Changelogen plugin', async () => {
+  const { counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert.deepEqual(counters, {
+    ...baseCounters,
+    devDependencies: 1,
```

Comment:
How is `changelog` used? I guess it can be referenced in a `package.json` script or in a GitHub Action workflow file? I think it would be good to reflect that in the fixture so we verify in the test the `changelog` dependency is not unused. We would also need to add a minimal `changelog` package to `node_modules` folder there.